float64 easting
float64 northing
float64 euclidDist
float64 thrust
float64 headingAdj
float64 desHead



